package.json

```diff
@@ -34,10 +34,13 @@
         "gulp-autoprefixer": "^7.0.1",
         "gulp-clean-css": "^4.2.0",
         "gulp-if": "^3.0.0",
-        "gulp-sass": "^4.0.2",
+        "gulp-sass": "^5.1.0",
         "gulp-sourcemaps": "^2.6.5",
         "lodash.debounce": "^4.0.8",
         "map-stream": "^0.0.7",
         "vinyl-fs": "^3.0.3"
+    },
+    "devDependencies": {
+        "sass": "^1.35.2"
```

Comment:
I could be wrong, but: Shouldn't this just be a regular dependency?

Reply:
Yep, this won't work if you don't already have the `sass` compiler installed. Try removing `node_modules` and doing a fresh install pointing `eleventy-plugin-sass` to your fork, you'll notice that it fails with the error: `Cannot find module 'sass'`